third_party/cpgfunctionEP-0.3.2/include/blas/blas.h

```diff
@@ -0,0 +1,26 @@
+//
```

Comment:
Two BLAS functions were ported to C++, `axpy` and `spmv`. 

axpy - takes the form `y = ax + y`
spmv - takes the form `y = Ax + y` where A is a packed symmetric matrix